app/views/notifications/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<p id="notice"><%= notice %></p>
+<div class='row'>
+  <div class='col'>
+    <h1>Notifications</h1>
+  </div>
+  <div class='col'>
+    <div class='float-right'>
+      <%#= link_to 'Reindex', reindex_parent_objects_path, method: 'post', class: 'btn btn-primary' %>
```

Comment:
Remove these commented lines?

Reply:
@orangewolf are there some links you wanted me to add here? Or can we delete these for now

Reply:
remove is fine for now